lib/chef/node/attribute_collections.rb

```diff
@@ -73,9 +73,7 @@ def to_yaml(*opts)
 
       def convert_value(value)
         case value
-        when VividMash
-          value
-        when AttrArray
+        when VividMash, AttrArray
```

Comment:
This code is repeated about 50 lines down.

Reply:
Fixed